go/runtime/bundle/manifest.go

```diff
@@ -124,20 +138,24 @@ func (m *Manifest) GetComponentByID(id component.ID) *Component {
 	return nil
 }
 
-// GetVersion returns the runtime version.
-func (m *Manifest) GetVersion() version.Version {
-	// We also support legacy manifests which define version at the top-level.
-	for _, comp := range m.Components {
-		if !comp.ID().IsRONL() {
-			continue
-		}
+// UnmarshalJSON customizes the unmarshalling of the manifest.
+func (m *Manifest) UnmarshalJSON(b []byte) (err error) {
+	// Unmarshal into the auxiliary struct to avoid recursion.
+	type Alias Manifest
```

Comment:
There should be no need for this type to be capitalized, private types should work too.